bsp/avr32uc3b0/SConscript

```diff
@@ -2,7 +2,7 @@ import rtconfig
 Import('RTT_ROOT')
 from building import *
 
-src_bsp = ['application.c', 'startup.c', 'board.c', 'drv_uart.c']
+src_bsp = ['application.c', 'startup.c', 'board.c', 'drv_uart.c', 'drv_gpio.c']
```

Comment:
Please use Kconfig and driver . Like STM32.
https://github.com/RT-Thread/rt-thread/blob/9dd1d73c94b55324229c42b4744fecab00086ae0/bsp/stm32/libraries/HAL_Drivers/SConscript#L10-L67

Reply:
Dear @supperthomas, many thanks for your suggestion. I have implemented something similar. Can you please check if this is alright?